paypalscript.py

```diff
@@ -1,16 +1,49 @@
 #!/usr/bin/env python3
+import argparse
 import csv
+import os.path
+
+#
+# Parse and set Input and Output Files
+#
+
+parser = argparse.ArgumentParser(description='Takes a PayPal CSV export file and generates a GnuCash formatted CSV import file.  Default input file name is Paypal Export.csv.  Defaults output file name is output.csv')
+parser.add_argument('-i','--input', help='Input file name',required=False)
+parser.add_argument('-o','--output',help='Output file name', required=False)
+args = parser.parse_args()
+
+inputFile = args.input
+if not bool(inputFile):
+    inputFile = 'Paypal Export.csv'
+
+if not os.path.exists(inputFile.strip('\n')):
+    print("Input file does not exist: " + inputFile)
+    quit()
```

Comment:
Set this as required in the arg and remove this check

Reply:
This is a check to see if the argument can be used to locate an existing file.  I don't believe argparse can check to see if the file exits on its own, but I'm new.

Reply:
This same check happens when you try to open it